src/engine/LoadingScreen.cs

```diff
@@ -31,9 +31,9 @@ public class LoadingScreen : Control
     private Label tipLabel;
     private Control spinner;
 
-    private string loadingMessage = "Loading";
+    private string loadingMessage = TranslationServer.Translate("LOADING_LOADING");
```

Comment:
I think that a generic LOADING string would be fine, I think we'll want to re-use some single words that are safe to use in multiple contexts.